bagel/cli.py

```diff
@@ -92,25 +92,24 @@ def pheno(
 
     # TODO: needs refactoring once we handle multiple participant IDs
     participants = column_mapping.get("participant")[0]
-    # TODO: handle if no session_ID column exists
-    session_column = column_mapping.get("session")[0]
+
+    # This is None if there is no session column
+    session_column = column_mapping.get("session")
 
     for participant in pheno_df[participants].unique():
-        # TODO: needs refactoring once we handle phenotypic information at the session level
-        # for the moment we are not creating any session instances in the phenotypic graph
-        # we treat the phenotypic information in the first row of the _sub_pheno dataframe
-        # as reflecting the subject level phenotypic information
         _sub_pheno = pheno_df.query(f"{participants} == '{str(participant)}'")
 
-        # TODO ensure we don't have duplicates in the session ID
-        session_names = _sub_pheno[session_column].unique()
-
         sessions = []
-        for session_name in session_names:
+        for session_row_idx, session_row in _sub_pheno.iterrows():
+            # If there is no session column, we create a session with a custom label "ses-nb01" to assign each subject's phenotypic data to
+            if session_column is None:
+                session_name = "ses-nb01"  # TODO: Should we make this more obscure to avoid potential overlap with actual session names?
```

Comment:
Yeah, I'm not too sure. I used "ses-nb01" since this is what we currently use for the custom BIDS session we create when there's no session layer, 
https://github.com/neurobagel/bagel-cli/blob/b66b9eee4ad70cbd3e46bc1c80115f3f82ea967f/bagel/cli.py#L284

but maybe switching to a non-BIDS compliant name would also be a good idea to avoid conflicts.

I think this would be worth addressing in a separate issue: https://github.com/neurobagel/bagel-cli/issues/262

If we want to rename this session, we should probably rename both the pheno + imaging one so we don't have two naming conventions floating around.